### DISCUSSION TITLE
netty depends on SNAPSHOT version of netty-tcnative-boringssl-static

### DISCUSSION THREAD

**Discussion by mwilso3** [netty/netty#14034] [General]
Curious why netty doesn't depend on a final version for netty-tcnative-boringssl-static? 

We bump our netty and tcnative versions in sync to avoid issues, however, we're currently unable to do so as netty 4.1.109.Final uses 2 versions of tcnative - 2.0.65.Final and 2.0.66.Final-SNAPSHOT for netty-tcnative-boringssl-static.

**Comment by normanmaurer** [comment]
@mwilso3 it only depends on the snapshot in a specific profile to allow us to test against snapshots during build. So I am not sure wha the problem is

**Reply by mwilso3** [reply]
Ah sorry. That's my bad, I completely missed the profile tag before the version.

Thanks for clarifying.
